src/nova-components/CreateFavorites/resources/js/components/Tool.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <div class="content">
```

Comment:
There's no class called `content` in tool.css, Remove undefined class names please